lib/govspeak.rb

```diff
@@ -302,7 +302,12 @@ def render_image(image)
     end
 
     extension("address", surrounded_by("$A")) do |body|
-      %(\n<div class="address"><div class="adr org fn"><p markdown="1">\n#{body.sub("\n", '').gsub("\n", '<br />')}\n</p></div></div>\n)
+      <<~BODY
```

Comment:
**praise:** heredocs

Thanks for the heredoc explainer. I didn't know they came with flavours